src/components/Campaign.tsx

```diff
@@ -53,19 +57,53 @@ export const Campaign = () => {
           <Header>
             <EditorTopBar
               data-testid={editorTopBarTestId}
-              onSave={save}
               title={
                 campaignContentQuery.data?.campaignName
                   ? campaignContentQuery.data.campaignName
                   : ""
               }
-            />
+            >
+              <ul className="ed-header-list">
+                <li>
+                  {campaignContentQuery.data?.type === "unlayer" ? (
+                    <button
+                      type="button"
+                      onClick={() => setOpen(true)}
+                      className="dp-button button-medium primary-green"
+                    >
+                      <FormattedMessage id="save_template" />
+                    </button>
+                  ) : (
+                    ""
+                  )}
+                </li>
+                <li>
+                  <button
+                    type="button"
+                    onClick={save}
+                    className="dp-button button-medium primary-green"
+                  >
+                    <FormattedMessage id="save" />
+                  </button>
+                </li>
+              </ul>
+            </EditorTopBar>
           </Header>
           <Footer>
             <EditorBottomBar {...continuationUrls}></EditorBottomBar>
           </Footer>
         </>
       )}
+      {campaignContentQuery.data?.type === "unlayer" ? (
+        <SaveAsTemplateModal
+          isOpen={isOpen}
+          content={campaignContentQuery.data}
+          defaultName={campaignContentQuery.data.campaignName}
```

Comment:
I have no strong opinion on this, but it could be changed as:

``` suggestion
          campaign={campaignContentQuery.data}
```

Reply:
I don't think so, because _campaignContentQuery.data_ is a **_campaignContent_** and _SaveAsTemplateModal_ must recibe a **_UnlayerContent_** also need the defaultName is for this reason that defaultName is a prop here.

The defaultName is default because could be change by user so in the <input name="name" /> the defaultName is a react defaultValue.

Reply:
👍